build.assets/tooling/cmd/build-os-package-repos/config.go

```diff
@@ -130,13 +130,13 @@ func NewConfigWithFlagset(fs *flag.FlagSet) *Config {
 	c.S3Config = NewS3ConfigWithFlagset(fs)
 
 	fs.StringVar(&c.artifactPath, "artifact-path", "/artifacts", "Path to the filesystem tree containing the *.deb or *.rpm files to add to the repos")
-	fs.StringVar(&c.artifactVersion, "artifact-version", "", "The version of the artifacts that will be added to the repos")
 	fs.Visit(func(f *flag.Flag) {
 		if f.Name == "-h" || f.Name == "--help" {
 			c.printHelp = true
 		}
 	})
 	fs.StringVar(&c.releaseChannel, "release-channel", "", "The release channel of the repos that the artifacts should be added to")
+	fs.StringVar(&c.versionChannel, "version-channel", "", "The version channel of the repos that the artifacts should be added to. Semver values will be truncated to major version.")
```

Comment:
Nit: Give a few examples of what a release channel or a version channel can be in the help messages.